backend/app.js

```diff
@@ -1,7 +1,49 @@
 const express = require("express");
 const app = express();
-const port = 3000;
+const port = 3001;
+const cors = require("cors");
+const fetch = require("node-fetch");
+import { userId, basicAuth } from "./config";
 
-app.get("/", (req, res) => res.send("Hello World!"));
+app.use(cors());
 
-app.listen(port, () => console.log(`Example app listening on port ${port}!`));
+app.get("/inventory", function(req, res) {
+  getId().then(result => res.send(result));
+});
+
+async function getId() {
+  let idList = [];
+
+  const response = await fetch(
+    "https://api.bigcartel.com/v1/accounts/" + userId + "/products",
+    {
+      headers: {
+        Authorization: "Basic " + basicAuth,
+        "Content-type": "application/vnd.api+json",
+        Accept: "application/vnd.api+json"
+      }
+    }
+  );
+  const json = await response.json();
+  async function getInventory() {
```

Comment:
cleaner to define this as a separate function that takes in a json parameter - nested functions I believe are not the common convention for JS